examples/addons.yaml

```diff
@@ -0,0 +1,750 @@
+# Calico Version v3.6
```

Comment:
Probably should update to 3.8 while we are adding manifests back: https://docs.projectcalico.org/v3.8/manifests/calico.yaml

Reply:
Updated 😊